tests/phpunit.integration.xml

```diff
@@ -25,6 +25,7 @@
 		 timeoutForSmallTests="900"
 		 timeoutForMediumTests="900"
 		 timeoutForLargeTests="900"
+		 cacheResult="true"
```

Comment:
That's only for localhost I guess.

Reply:
We have the cache file in .gitignore anyway. It doesn't make a difference for CI as the file doesn't exist there.

Reply:
Turns out this is even `true` by default now https://phpunit.readthedocs.io/en/9.3/configuration.html